src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,243 @@
+
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0        
+
+    ppa_record_1, pal_record_1 = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                [pal_record],
+                                                                pal_low_frequency,
+                                                                pal_high_frequency,
+                                                                'test_user_1')
+    # Load DPA record
+    dpa_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'dpa_0.json')))
+    dpa_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'dpa_1.json')))
+    dpa_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'dpa_0.json')))
+
+    # Registration and grant records for multiple iterations
+    cbsd_records_iter_1_domain_proxy_0 = {
+        'registrationRequests': [device_1, device_2],
+        'grantRequests': [grant_request_1, grant_request_2],
+        'conditionalRegistrationData': conditionals
+    }
+    cbsd_records_iter_1_domain_proxy_1 = {
+        'registrationRequests': [device_3],
+        'grantRequests': [grant_request_3],
+        'conditionalRegistrationData': {}
+    }
+    cbsd_records_iter_2_domain_proxy_0 = {
+        'registrationRequests': [device_4],
+        'grantRequests': [grant_request_4],
+        'conditionalRegistrationData': {}
+    }
+    cbsd_records_iter_2_domain_proxy_1 = {
+        'registrationRequests': [device_5, device_6],
+        'grantRequests': [grant_request_5, grant_request_6],
+        'conditionalRegistrationData': {}
+    }
+    
+    # Protected entities records for multiple iterations
+    protected_entities_iter_1 = {
+        'palRecords': [pal_record_1],
+        'ppaRecords': [ppa_record_1], 
+        'escRecords': [esc_record_1]
+    }
+    protected_entities_iter_2 = {
+        'gwpzRecords': [gwpz_record_1], 
+        'fssRecords': [fss_record_1]
+    }
+
+    # SAS Test Harnesses configurations, 
+    # Following configurations are for two SAS test harnesses for two iterations
+    sas_test_harness_device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    sas_test_harness_device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    sas_test_harness_device_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    sas_test_harness_device_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    sas_test_harness_device_5 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    sas_test_harness_device_6 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_f.json')))
+
+    # Generate Cbsd FAD Records for SAS Test Harness 0, iteration 0
+    cbsd_fad_records_sas_test_harness_0_iteration_0 = generateCbsdRecords([sas_test_harness_device_1], [[grant_request_1]])
```

Comment:
Recommend you unify your naming convention here with the one used for DPs above: `cbsd_records_iter_1_domain_proxy_0` uses iter # then DP #, whereas here you use TH # then iter #.